### ISSUE TITLE
Does not cooperate with shell pipes.

### ISSUE THREAD

**Issue by sydneypemberton1986** [auntbertha/astdiff#2]
I love this tool! But I would like to be able to use it with pipes.

For example 

`astdiff 1>/dev/null | errors.txt`

Somehow this tool is breaking out of this convention making this not possible.

**Comment by waltermoreira** [comment]
Good point. We can definitely do a better job in splitting stdout and stderr. We might also need an option to disable the ANSI output, so it can be pure text usable in a pipe.
